/* Copyright 2023 The MediaPipe Authors. All Rights Reserved.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
==============================================================================*/

syntax = "proto2";

package mediapipe.tasks.vision.face_landmarker.proto;

import "mediapipe/framework/calculator.proto";
import "mediapipe/framework/calculator_options.proto";

message TensorsToFaceLandmarksGraphOptions {
  extend mediapipe.CalculatorOptions {
    optional TensorsToFaceLandmarksGraphOptions ext = 509621260;
  }

  // Whether the landmarks model is with attention on lips and eyes. Attention
  // provides more accuracy on lips and eye regions as well as iris landmarks.
  optional bool is_attention_model = 1 [default = false];

  optional int32 input_image_width = 2;

  optional int32 input_image_height = 3;
}
